firmware/controllers/trigger/trigger_emulator_algo.cpp

```diff
@@ -148,7 +149,9 @@ static void emulatorApplyPinState(int stateIndex, PwmConfig *state) /* pwm_gen_c
 	}
 
 #if EFI_PROD_CODE
-	applyPinState(stateIndex, state);
+	if (hasStimPins) {
```

Comment:
we need code comments explaining the reasons behind new conditional logic

Reply:
I figured the logic
```
if (has_pins) {
    use_pins();
}
```
was pretty straight forward, but sure